website/content/commands/server-install.mdx

```diff
@@ -135,6 +135,7 @@ and disable the UI, the command would be:
 - `-nomad-csi-volume-capacity-min=<int>` - Nomad CSI volume capacity minimum, in bytes. The default is 1073741824.
 - `-nomad-csi-volume-capacity-max=<int>` - Nomad CSI volume capacity maximum, in bytes. The default is 2147483648.
 - `-nomad-csi-fs=<string>` - Nomad CSI volume mount option file system. The default is xfs.
+- `-nomad-csi-mount-flags=<string>` - Nomad CSI volume mount option flags. The default is noatime. The default is noatime.
```

Comment:
```suggestion
- `-nomad-csi-mount-flags=<string>` - Nomad CSI volume mount option flags. The default is noatime.
```

Reply:
Oops, thought i had removed that before pushing! Good catch and fixed